guides/v2.3/extension-dev-guide/routing.md

```diff
@@ -165,6 +165,20 @@ $this->_forward('action', 'controller', 'Other_Module')
 {: .bs-callout .bs-callout-tip }
 Use the [`ActionFactory`] in your router to create an instance of an `Action` class.
 
+{: .bs-callout .bs-callout-info }
+Action class should return a `result object`.
+
+## Result object
+
+Name | Description
+--- | ---
+`json` | Sets `Content-Type:application/json` in the header and returns a json encoded representation of an array with data.
+`raw` | Returns the data as it's been set. Does not set a `Content-Type` of the header!
```

Comment:
Remove ending exclamation mark

Reply:
**done**

@erikmarr , could you please review?

Thank you!